### PR TITLE
Add showEditMenuOnSelectRange flag

### PR DESCRIPTION
**Pull request by GiacomoPignoni**
PR for #254 

I've simply added a flag and use it when selecting a range to choose if to show the edit menu or not

### CODE REVIEW

File: Sources/Runestone/TextView/Core/TextView.swift [hunk 1]
```diff
@@ -568,6 +568,8 @@ open class TextView: UIScrollView {
             textInputView.lineEndings = newValue
         }
     }
+    /// If this flag is enable the TextView will present the edit menu when select a range 
+    public var showEditMenuOnSelectRange: Bool = true
```
Comment (simonbs) [inline Sources/Runestone/TextView/Core/TextView.swift:572]: Can we rename this to make it clear that it's related to highlighted ranges? Something like the following, maybe. It's long and boring but it's descriptive 😄

```suggestion
    public var showMenuAfterNavigatingToHighlightedRange = true
```

File: Sources/Runestone/TextView/Core/TextView.swift [hunk 2]
```diff
@@ -568,6 +568,8 @@ open class TextView: UIScrollView {
             textInputView.lineEndings = newValue
         }
     }
+    /// If this flag is enable the TextView will present the edit menu when select a range
```
Comment (simonbs) [inline Sources/Runestone/TextView/Core/TextView.swift:571]: Can we change this to be slightly more descriptive?

```suggestion
    /// When enabled the text view will present a menu with actions actions such as Copy and Replace after navigating to a highlighted range.
```

**Comment by GiacomoPignoni** [comment]
Sorry for the late answer.
For sore the name and the comment now look better. 

I've tried to use the automatic commit to resolve your suggestions, so I’m not sure if I really did it 🤔

**Comment by simonbs** [comment]
@GiacomoPignoni Thanks for merging the changes in. It seems like SwiftLint is failing. Will you have a look at that? 😊

**Pull request closed without merging**